packages/puppeteer-core/src/common/bidi/Serializer.ts

```diff
@@ -112,8 +126,25 @@ export class BidiSerializer {
     }
   }
 
-  static serialize(arg: unknown): Bidi.CommonDataTypes.LocalOrRemoteValue {
+  static serialize(
+    arg: unknown,
+    context: Page
+  ): Bidi.CommonDataTypes.LocalOrRemoteValue {
     // TODO: See use case of LazyArgs
+    const objectHandle = arg && arg instanceof JSHandle ? arg : null;
+    if (objectHandle) {
+      // TODO:
```

Comment:
what is this TODO about?

Reply:
It was for the execution context match, at the time it was not implemented removing